topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.md

```diff
@@ -0,0 +1,564 @@
+
+<div class="Step-by-step" markdown="1">
+> <hands-on-title> Identify contamination </hands-on-title>
+>
+> 1. {% tool [Recentrifuge](toolshed.g2.bx.psu.edu/repos/iuc/recentrifuge/recentrifuge/1.12.1+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Select taxonomy file tabular formated"*: **Classification** output of **Krancken2** {% icon tool %}
+>    - *"Type of input file (Centrifuge, CLARK, Generic, Kraken, LMAT)"*: `Kraken`
+>    - In *"Database type"*:
+>        - *"Cached database whith taxa ID"*: `NCBI-2023-06-27`
+>    - In *"Output options"*:
+>        - *"Type of extra output to be generated (default on CSV)"*: `TSV`
+>        - *"Remove the log file"*: `Yes`
+>    - In *" Fine tuning of algorithm parameters"*:
+>        - *"Strain level instead of species as the resolution limit for the robust contamination removal algorithm; use with caution, this is an experimental feature"*: `Yes`
+>
+{: .hands_on}
+</div>
+
+**Recentrifuge** generates 3 outputs:
+
+- A **statistic table** with general statistics about assignations
+
+   > <question-title></question-title>
+   >
+   > 1. How many sequences have been used?
+   > 2. How many sequences have been classified?
+   >
+   > > <solution-title></solution-title>
+   > > 1. 451,780
+   > > 2. 450,715
+   > {: .solution}
+   {: .question}
+
+- A **data table** with a report for each taxa
+
+   > <question-title></question-title>
+   >
+   > 1. How many taxa have been kept?
+   > 2. What is the lowest level in the data?
+   >
+   > > <solution-title></solution-title>
+   > > 1. 187 (190 lines including 3 header lines)
+   > > 2. The lowest level is strain.
+   > {: .solution}
+   {: .question}
+
+- A **HTML report** with a Krona chart
+
+   <iframe id="recentrifuge" src="recentrifuge_report.html" frameBorder="0" width="100%" height="900px"> ![Krona chart with multi-layered pie chart representing the community profile with in the center the higher taxonomy levels (i.e. domain) and on the exterior the more detailed ones (i.e. species)](./images/recentrifuge.png) </iframe>
+
+   > <question-title></question-title>
+   >
+   > 1. What is the percentage of classified sequences?
+   > 2. When clicking on *Staphylococcus aureus*, what can we say about the strains?
+   > 3. Is there any contamination?
+   >
+   > > <solution-title></solution-title>
+   > >
+   > > 1. 99.8%
+   > > 2. 99% of sequences assigned to *Staphylococcus aureus* are not assigned to any strains, probably because they are too similar to several strains. *Staphylococcus aureus* subsp. aureus JKD6159 is the strain with the most classified sequences, but only 0.3% of the sequences assigned to *Staphylococcus aureus*.
+   > > 3. There is no sign of a possible contamination. Most sequences are classified to taxon on the *Staphylococcus aureus* taxonomy. Only 3% of the sequences are not classified to *Staphylococcus*.
+   > >
+   > {: .solution}
+   >
+   {: .question}
+
```

Comment:
Once we have identified contamination, what should we do? We could 1-2 sentences to explain the next step